<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1179</width>
    <height>1709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>悦谱转图比对系统 EPVS-V1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1171</width>
      <height>821</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabMainFileExplorer">
     <attribute name="title">
      <string>文件管理</string>
     </attribute>
     <widget class="QWidget" name="widget_fileExplorer_top" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>1141</width>
        <height>31</height>
       </rect>
      </property>
      <widget class="QLineEdit" name="lineEditMainFileExplorerSearch">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>0</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonMainFileExplorerForward">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>0</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>前进</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonMainFileExplorerUp">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>0</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>上级</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonMainFileExplorerBack">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>后退</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxMainFileExplorerPath">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>0</y>
         <width>601</width>
         <height>20</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_fileExplorer_bot" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>59</y>
        <width>1141</width>
        <height>751</height>
       </rect>
      </property>
      <widget class="QWidget" name="widgetMainFileExplorerRightMain" native="true">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>0</y>
         <width>881</width>
         <height>781</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="widgetMainFileExplorerSideBar" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>251</width>
         <height>781</height>
        </rect>
       </property>
       <widget class="QWidget" name="widgetLeftSiderTop" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>231</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="widgetLeftSiderBot" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>340</y>
          <width>231</width>
          <height>351</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabMainEPVS">
     <attribute name="title">
      <string>转图比对</string>
     </attribute>
     <widget class="QWidget" name="widget_vs_left" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>901</width>
        <height>821</height>
       </rect>
      </property>
      <widget class="QWidget" name="widget_vs_left_top" native="true">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>901</width>
         <height>151</height>
        </rect>
       </property>
       <widget class="QGroupBox" name="groupBoxJobA">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>0</y>
          <width>278</width>
          <height>139</height>
         </rect>
        </property>
        <property name="title">
         <string>料号A</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QPushButton" name="pushButtonInputA">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>50</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonImportA">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>50</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelStatusJobA">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>171</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>状态：</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonJobAReset">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>100</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>重置A</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBoxJobB">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>0</y>
          <width>311</width>
          <height>139</height>
         </rect>
        </property>
        <property name="title">
         <string>料号B</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QPushButton" name="pushButtonImportB">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>50</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonInputB">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>50</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelStatusJobB">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>161</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>状态：</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonJobBReset">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>100</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>重置B</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="widget_vs_left_top_compare" native="true">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>-1</y>
          <width>120</width>
          <height>151</height>
         </rect>
        </property>
        <widget class="QPushButton" name="pushButtonAllReset">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>101</y>
           <width>116</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>重置所有</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonVS">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>16</y>
           <width>116</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>比图</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxVSMethod">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>60</y>
           <width>116</width>
           <height>20</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>方案1：G比图</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>方案2：悦谱比图</string>
          </property>
         </item>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_vs_left_bot" native="true">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>149</y>
         <width>901</width>
         <height>661</height>
        </rect>
       </property>
       <widget class="QTableWidget" name="tableWidgetVS">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>891</width>
          <height>641</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_vs_right" native="true">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>0</y>
        <width>221</width>
        <height>821</height>
       </rect>
      </property>
      <widget class="QWidget" name="widget_vs_right_top" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>221</width>
         <height>151</height>
        </rect>
       </property>
       <widget class="QPushButton" name="pushButtonHelp">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>90</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>帮助</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonLoadEPCAM">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>75</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>加载EPCAM</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonSettings">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>50</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>配置</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonSave">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>保存结果</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_vs_right_bot" native="true">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>159</y>
         <width>221</width>
         <height>651</height>
        </rect>
       </property>
       <widget class="QTextBrowser" name="textBrowserMain">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>221</width>
          <height>641</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1179</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
